roles/create_vms/templates/create_vm.sh.j2

```diff
@@ -32,6 +32,9 @@ virt-install \
     --features smm=on \
     {% endif %}
     {% endif %}
+    {% if additional_virt_install_options %}
+    {{ additional_virt_install_options }}
+    {% endif%}
```

Comment:
Nit: can you please add a white space right after the endif? 
so it would be    ` {% endif %}`
other than that, lgtm.